### ISSUE TITLE
Teensy 4.1 support

### ISSUE THREAD

**Issue by sp4xkb** [g4klx/MMDVM#332]
Teensy 4.1 support

Will you update the sources for Teensy 4.1?

Regards

**Comment by g4klx** [comment]
No. The 4.1 doesn’t have the hardware required. It has no ADC or DAC and that rules it out.
Sorry
Jonathan G4KLX 


Sent from Yahoo Mail for iPhone


On Wednesday, December 21, 2022, 21:35, sp4xkb ***@***.***> wrote:




Teensy 4.1 support

Will you update the sources for Teensy 4.1?

Regards
